### PR TITLE
Cleanup and fullname

### PR DESCRIPTION
**Pull request by reist**
Adding Amir to reviewers for his bash mastery

### CODE REVIEW

**Comment by amiryalsa** [comment]
Why do we need Jenkins to support giving the full file name? Why can’t we give just the version like it currently expects?

**Comment by reist** [comment]
> Why do we need Jenkins to support giving the full file name? Why can’t we give just the version like it currently expects?

Because the files (and versions) are in S3, and @nmagedman stated it's easier to copy the full file name from there than to mark just the version.

File: Jenkinsfile.sh [hunk 1]
```diff
@@ -17,9 +17,16 @@ export ENV_ARG=$environment
 export RC_DIR_ARG="/opt/rocket-chat"
 export S3_BUCKET_ARG="seekingalpha-rocketchat-builds"
 
+if [[ "$version" == rocket.chat-*.tgz ]]
+then
+  versionized_file="$version"
```
Comment (nmagedman) [inline Jenkinsfile.sh]: Rename this variable to `RC_TARBALL_ARG`

* Line 14 defines a convention: Template variables should end in `_ARG`.
* Implied by that is that the variables are in ALL_CAPS
* "versionized file" tells me how it was _generated_, but not what _it is_.  It's a **tarball**.

File: Jenkinsfile.sh [hunk 2]
```diff
@@ -17,9 +17,16 @@ export ENV_ARG=$environment
...
+else
+  versionized_file="rocket.chat-$version.tgz"
+fi
+
 ## Render Script Templates
 ## Note: $version is a Jenkins job parameter
```
Comment (nmagedman) [inline Jenkinsfile.sh]: Move this comment either to line 20 (where `$version` is used) or to the top of the file (describing its inputs)

**Review by nmagedman — CHANGES_REQUESTED** [review]

**Comment by nmagedman** [comment]
> Why do we need Jenkins to support giving the full file name? Why can’t we give just the version like it currently expects?

@amiryalsa , I specifically requested this change in the description of [AR-2692](https://seekingalpha.atlassian.net/browse/AR-2692):

> Support a $version string of the full tarball name: rocket.chat-VERSION.tgz.  I find that I am usually copying the VERSION string from the full fillename anyway, so it is easier to double-click and highlight the full name.

That is, _in practice_, when I launch the Jenkins job for this, I always do the following:
* $ aws s3 ls s3://seekingalpha-rocketchat-builds/
* Find the tarball I want to deploy
* Copy/paste the "version" part of the filename.

To make my life easier, I want to double-click the filename (highlighting the whole thing) rather than having to carefully highlight just the part of the filename that I want.

**Review by nmagedman — APPROVED** [review]

**Pull request merged**